### DISCUSSION TITLE
Update related record

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17217] [How to]
Hello,
YF 6.4
I try to get the number of opportunities related to a campaign x through a workflow Every time a record is saved (eg create, modify) / Update related record. create a text field in opportunities and the value 1 is assigned to ca opportunities and I try to get the number of opportunities assigned in a campaign x in the No. of Opportunities field, as I show below but so far I have not achieved anything, I have Tried various ways with no results.
In campaign module
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/29c5b3ea-0dc5-4786-9416-db2cd389956a)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/fe81fd52-9d57-4ca4-aa96-5c8d4d154c88)

in opportunities
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/11f9d2f8-d1a0-4610-8620-ebbb889f8035)
Workflows
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/b45d786b-c1e9-4379-8b3c-3b16462c033d)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/612e481e-dfe7-4bf7-a146-a7b7c31c9756)

**Comment by apcloic** [comment]
@krseguros I don't think there is a built-in expression for this but you could use a custom function.
#16719
#13231
